src/bci_build/package/postgres/README.md.j2

```diff
@@ -24,6 +24,10 @@ $ podman run -it --rm -p 5432:5432 -e POSTGRES_PASSWORD=my-password -v /path/to/
 
 PostgreSQL data directory location.
 
+**Note 1:** `initdb` requires this location to be empty to create a new database.
+
+**Note 2:** If the volume points to a file system mount point, a remote folder that can't be chowned to the `postgres` user, or a location that already contains files (including `lost+found` and dotfiles), a new subdirectory must be created within the select storage method to contain the PostgreSQL data.
```

Comment:
```suggestion
**Note 2:** If the volume points either to a file system mount point, a remote folder that cannot be owned by the `postgres` user, or a location that already contains files (including `lost+found` and dotfiles), a new subdirectory for storing the PostgreSQL data must be created within the selected storage method.
```

Reply:
_within the selected storage method_ sounds strange to me. Can you really create a subdir **within** a storage method?

Reply:
Within in this case is inside, switching to inside sounds odd to me:

> a new subdirectory for storing the PostgreSQL data must be created inside the selected storage method.

The issue in note 2 is the following:

- Inside the container all PSQL sees is a folder, PSQL is unaware that this is a volume and neither what is behind this volume
- PSQL can't use the current folder because it has some issue (e.g. not empty location)
- For PSQL, the solution is to mount something as in PGDATA
- For the user, one solution is to create a new sub folder in the volume they are trying to use

The idea here is to document that certain things can't be used as PGDATA, we don't really need to provide the solution, as the solution could vary per setup.

The upstream Docker Hub image uses a similar text.

Reply:
@dmpop does my previous message explains within?

Reply:
can we simply say "within the PGDATA volume"?